identity/claim.go

```diff
@@ -20,39 +20,25 @@
 package identity
 
 import (
-	"fmt"
-	"sync"
+	"github.com/ethereum/go-ethereum/crypto"
 )
 
-type Store interface {
-	Load(name string) (*Identity, error)
-	Save(*Identity) error
+// Claim structure
+type Claim struct {
+	Issuer    string `json:"issuer"`
+	Holder    string `json:"holder"`
```

Comment:
This should be "subject" rather than "holder" to be inline with the [verifiable claims user roles](https://w3c.github.io/vc-use-cases/#user-roles).

Reply:
I modified that due to @lukehedger request. 
"Could you rename `key`/`value` to `claim`/`signature`, `submitter` to `issuer` and `subject` to `holder`? This is closer to the naming conventions found in some standards."
revert back ?

Reply:
@orenyodfat - yes please, revert `holder` back to `subject`. The others are OK.